components/custom/club-card.vue

```diff
@@ -57,9 +57,11 @@ const props = defineProps({
       </div>
     </CardContent>
     <CardFooter>
-      <Button class="w-full">
-        详细信息
-      </Button>
+      <nuxt-link :to="`/clubs/${props.club.groups[0].C_GroupsID}`">
```

Comment:
1. Use `NuxtLink` for better code style. 
2. Add a `w-full` class to expand the width.